routers/web/repo/pull.go

```diff
@@ -1457,6 +1457,18 @@ func CompareAndPullRequestPost(ctx *context.Context) {
 		return
 	}
 
+	if projectID > 0 {
+		if !ctx.Repo.CanRead(unit.TypeProjects) {
```

Comment:
Is `read projects` the correct permission for this?

Reply:
It's not right. It should be write permission. Issue permission is special.

Reply:
Okay, I changed it to write